exasol/python_extension_common/deployment/language_container_deployer.py

```diff
@@ -162,6 +162,9 @@ def run(self, container_file: Optional[Path] = None,
         allow_override   - If True the activation of a language container with the same alias will be
                            overriden, otherwise a RuntimeException will be thrown.
         wait_for_completion - If True will wait until the language container becomes operational.
+                            For this to work either of the two conditions should be met.
+                            The pyexasol connection should have an open schema, or
```

Comment:
This means, we need to set the schema for the Pyexasol connection in the init of TE and SME. Alternatively, we would need to add an additional parameter schema.

Reply:
Do you think we need to provide this option? For the AI-Lab, TE, and SME it doesn't matter. They use cli, where we do have schema as a parameter. In pytest plugins we open the schema, but we should have the permissions to create one anyway.